framework/source/class/qx/test/locale/Number.js

```diff
@@ -0,0 +1,67 @@
+/* ************************************************************************
+
+  qooxdoo - the new era of web development
+
+  http://qooxdoo.org
+
+  Copyright:
+    2007-2008 1&1 Internet AG, Germany, http://www.1und1.de
+
+  License:
+    MIT: https://opensource.org/licenses/MIT
+    See the LICENSE file in the project's top-level directory for details.
+
+  Authors:
+    * Romain Julian (rommni)
+
+************************************************************************ */
+
+qx.Class.define("qx.test.locale.Number", {
+  extend : qx.dev.unit.TestCase,
+
+  members: {
+    testNumberSeparators : function() {
+      // test not designed for generator build
+      if (!qx.core.Environment.get("qx.compilerVersion")) {
+        return;
+      }
+
+      var Number = qx.locale.Number;

```

Comment:
Overwriting the native Number object is certainly legal, but is it a good idea? Maybe use a different symbol?

Reply:
Hum effectively i took the idea from the qx.test.locale.Date, i will change that.